pkg/plugins/transform_plugin.go

```diff
@@ -77,7 +77,9 @@ type TransformWrapper struct {
 
 func (tw *TransformWrapper) Transform(ctx context.Context, query *tsdb.TsdbQuery) (*tsdb.Response, error) {
 	pbQuery := &pluginv2.QueryDataRequest{
-		Config:  &pluginv2.PluginConfig{},
+		PluginContext: &pluginv2.PluginContext{
+			// TODO: Things probably
```

Comment:
This looks interesting :smiley: No blocker for merging this, but lets discuss tomorrow

Reply:
Aye, will get to looking at this probably when I look into updating gel to work with this.